templates/arc/.gitignore

```diff
@@ -1,10 +1,8 @@
-.DS_Store
 node_modules
 
 /.cache
-/server/index.mjs
-/server/index.mjs.map
 /public/build
+/server
```

Comment:
The file config.arc is under version control in the /server directory. If you want to ignore the whole server directory, then I suggest that you move the `@aws` section to app.arc and remove config.arc.